src/js/jsx/sections/style/Type.jsx

```diff
@@ -506,26 +517,53 @@ define(function (require, exports, module) {
                     fontWeight: this._getCSSFontWeight(styleTitle) || 400
                 };
 
-                if (colorTiny) {
-                    typeStyle.color = colorTiny.toRgbString();
-                }
+                if (familyName === strings.STYLE.TYPE.MIXED) {
+                    var fillStyle = {
+                        height: "100%",
+                        width: "100%",
+                        backgroundColor: colorTiny ? colorTiny.toRgbString() : "transparent"
+                    };
 
-                return (
-                    <div
-                        className="type__preview"
-                        style={typeStyle}>
-                        Aa
-                    </div>
-                );
+                    return (
+                        <div
+                            className="fill__preview"
+                            style={fillStyle}/>
+                    );
+                } else {
+                    typeStyle = {
```

Comment:
Looks like `typeStyle` is only used in this scope and its initial value will always get overwritten. Maybe consider remove the outer `typeStyle` assignment?